crates/wdk-build/src/bindgen.rs

```diff
@@ -26,7 +26,7 @@ impl BuilderExt for Builder {
     ///
     /// Will return `wdk_build::ConfigError` if any of the resolved include or
     /// library paths do not exist
-    fn wdk_default(c_header_files: Vec<&str>, config: Config) -> Result<Self, ConfigError> {
+    fn wdk_default(c_header_files: Vec<&str>, config: &Config) -> Result<Self, ConfigError> {
```

Comment:
Haha. I made a similar change to this in a feature branch I haven't pr'ed yet

Reply:
Great minds think alike!